src/Util/ColumnParser.php

```diff
@@ -191,7 +190,6 @@ public function getType($field, $type)
         $collection = new Collection($reflector->getConstants());
 
         $validTypes = $collection->filter(function ($value, $constant) {
-            $value;
```

Comment:
Yeah just stating why I initially added that in case anyone was confused. I don't care either way :)